.github/workflows/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,53 @@
+name: 🐞 Bug
+description: File a bug/issue
+labels: [Bug, Needs Triage]
```

```suggestion
labels: [bug, needs triage]
```
I will need to create the label `needs triage` upon acceptance of PR.

---

Makes sense! I'll remove `Needs Triage` from the list. That also means that I've to add a `labels.yml` file for convenience, correct? If yes, then can you please suggest me a few labels?

---

Interesting, I didn't know you could create a `labels.yml` file. I wonder if there is a way to export the [labels that we have now in this repo](https://github.com/hyperledger-labs/documentation-template/labels).

---

@tkuhrt I've pushed the requested changes in the most recent [commit](https://github.com/hyperledger-labs/documentation-template/pull/10/commits/61df46217a3daa038a9aa32f115e66f2953343a4). For maintaining transparency, I've used the color codes of these [colors](https://twitter.com/github/status/954467266434945024) for the labels. Besides, I've added two more labels (`awaiting response` and `stale`) in the list of labels.

Do lemme know if any other changes are required :)